### DISCUSSION TITLE
Popular websites using `axios`

### DISCUSSION THREAD

**Discussion by spalt08** [axios/axios#5212] [Show and tell]
So far, I've only parsed ~10K most popular websites, but the number will be increased in the nearest future. Also, the detection works only for webpack bundles with ~70-90% accuracy.

<img width="1551" alt="image" src="https://user-images.githubusercontent.com/5869473/199058019-eb612b35-5e59-46f4-b9f1-e7985623705f.png">

Source: https://gradejs.com/package/axios

I'd like to share a tool I made for package maintainers that helps to discover real-world usage and version distribution for an NPM package.

I'd be happy to receive some feedback & thoughts from maintainers.

**Comment by jasonsaayman** [comment]
Awesome this looks super cool. I will check out the tool
